docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -6,7 +6,7 @@ The @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ RPC allows y
 !!!
 
 ## Checking if Dial Number is Available
-@![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ is a newer RPC, so there is a possibility that not all head units will support it. To find out if the RPC is supported by the head unit, check the system capability manager's @![iOS]`hmiCapabilities.phoneCall`!@ @![android,javaSE,javaEE]`hmiCapabilities.isPhoneCallAvailable()`!@ property after the manager has been started successfully.
+@![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ is a newer RPC, so there is a possibility that not all head units will support it. Once you have successfully connecting to the module, you can access the phone capability property on the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ instance.
```

Comment:
When you say **full property access** do you mean like this: `SDLManager.systemCapabilityManager.subscribe(capabilityType:updateHandler:)` ? I was trying to keep it simple since the code sample is below.

Not sure what needs to be checked using the Permission Manger.

Reply:
Sorry, this was from an earlier comment before I understood what you were doing.

However, I have a few other notes:
1. Before calling `subscribe`, the dev ought to call `isCapabilitySupported`.
2. The permission manager part would be to check if the dev actually _has permission_ to use `DialNumber`. I thought that it could also be used to check if `DialNumber` is supported to begin with. Your current method will work on some HUs, but between v3.0 (when `DialNumber` was introduced), and v4.5 (when `GetSystemCapability` was introduced), it will not.